ui/server/prisma/migrations/20220202110646_collection_member_add_has_joined_col/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "CollectionMember" ADD COLUMN     "hasJoined" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
This is causing all existing members to have `hasJoined` default to `false`. The subsequent migration won't change that since all fields will already be set from this migration. You can delete both migration files and generate one again, that have it set the default to `true`

Reply:
Alright, I'll do this